apps/api-express/src/app/daos/search.ts

```diff
@@ -46,11 +47,17 @@ export async function findQueryResults(
     );
     allParagraphsStandalone.push(...standaloneParagraphs);
   });
-  // sort in descending order
+  // Sort in descending order and remove empty items
   allParagraphsStandalone.sort(
     (a, b) => b.correlationScore - a.correlationScore
   );
-  return allParagraphsStandalone.slice(
+  const allParagraphsStandaloneFiltered = allParagraphsStandalone.filter(
+    (paragraph) =>
+      paragraph.body &&
+      paragraph.body !== '' &&
+      paragraph.body.trim().length > 0
```

Comment:
the empty string is falsy, so the first 2 conditions are synonymous.  I think you could boil this down to 

```
x.filter(paragraph => paragraph.body?.trim().length > 0)
```